static/Initialize-RootTerraformInfrastrucutre.ps1

```diff
@@ -5,29 +5,30 @@
 # Latest update: 2024-08-24
 [CmdletBinding()]
 param (
-    [Parameter()]
     [Parameter(Mandatory = $true)]
     [string]
-    $TenantId = "",
+    $TenantId,
     [Parameter(Mandatory = $true)]
     [string]
-    $SubscriptionId = "",
+    $SubscriptionId,
     [string]
-    $ResourceGroupName = "rg-infrastrucutre-management",
+    $ResourceGroupName = "rg-infrastructure-management",
     [string]
     $Location = "West Europe",
     [string]
-    $StorageAccountName = "cafterraformstate",
+    $StorageAccountName = "infrastructuremanagementstate",
```

Comment:
is the name not too long? i would suggest just `infrastructurestate` or something along the lines of `managementtfstate`